dfs/nfs/mounts_test.go

```diff
@@ -11,7 +11,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// +build unit
+// +build unit, linux
```

Comment:
The point is to be able to do something on OSX as opposed to today's status quo, which is nothing.  There's no plan to include OSX in the PR or nightly builds so breakages will occur.
Even though breakages will occur, and even though several subsystems will not compile or test, there are several areas where someone could make some progress on some issues without the need to VPN into a Linux machine.

For instance, one could make great head way on across-the-board replacement of glog, or replacement of codegansta, or standardizing tests on gochk.  The work wouldn't be definitive or final until it could be built/tested on Linux, but again, it is more than could be done today on OSX.